src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -493,13 +494,19 @@ public HttpResponseMessage GetNiceUrl(Udi id)
                 Mapper.Map<IContent, ContentItemBasic<ContentPropertyBasic>>(content,
                     opts =>
                     {
+                        if(string.IsNullOrEmpty(cultureName) == false)
+                        {
+                            opts.AfterMap((source, target) => target.Name = source.GetCultureName(cultureName));
```

Comment:
Hrm, we don't really use this type of mapping anywhere else in the codebase. For these culture specirfic things we normally map by passing in the culture to the mapper and then extracting the culture during the mapping process. I can show you a few examples. I'd prefer to do it that way then inlining the mapping logic.